cypress/integration/dashboard.js

```diff
@@ -0,0 +1,143 @@
+import faker from "faker";
+
+import Channels from "../apiRequests/Channels";
+import Customer from "../apiRequests/Customer";
+import { DASHBOARD_SELECTORS } from "../elements/dashboard/dashboard-selectors";
+import { HEADER_SELECTORS } from "../elements/header/header-selectors";
+import OrdersUtils from "../utils/ordersUtils";
+import ProductsUtils from "../utils/productsUtils";
+import ShippingUtils from "../utils/shippingUtils";
+
+// <reference types="cypress" />
+describe("User authorization", () => {
+  const startsWith = "Cy-";
+
+  const customer = new Customer();
+  const channels = new Channels();
+  const productsUtils = new ProductsUtils();
+  const shippingUtils = new ShippingUtils();
+  const ordersUtils = new OrdersUtils();
+
+  before(() => {
+    cy.clearSessionData().loginUserViaRequest();
+    customer.deleteCustomers(startsWith);
+    shippingUtils.deleteShipping(startsWith);
+    productsUtils.deleteProducts(startsWith);
+    channels.deleteTestChannels(startsWith);
+  });
+
+  beforeEach(() => {
+    cy.clearSessionData().loginUserViaRequest();
+  });
+
+  it("should all elements be visible on the dashboard", () => {
+    cy.visit("/")
+      .softAssertVisibility(DASHBOARD_SELECTORS.sales)
+      .softAssertVisibility(DASHBOARD_SELECTORS.orders)
+      .softAssertVisibility(DASHBOARD_SELECTORS.activity)
+      .softAssertVisibility(DASHBOARD_SELECTORS.topProducts)
+      .softAssertVisibility(DASHBOARD_SELECTORS.ordersReadyToFulfill)
+      .softAssertVisibility(DASHBOARD_SELECTORS.paymentsWaitingForCapture)
+      .softAssertVisibility(DASHBOARD_SELECTORS.productsOutOfStock);
+  });
+
+  it("should correct amount of orders be displayed", () => {
+    const randomName = startsWith + faker.random.number();
+    const randomEmail = randomName + "@example.com";
+    const randomNameProductOutOfStock = `${startsWith}${faker.random.number()}`;
+    const shippingPrice = 12;
+    const productPrice = 22;
+    const sales = productPrice * 2 + shippingPrice;
+
+    // Create channel, customer, product - everything needed to create order
+    cy.fixture("addresses").then(json => {
+      channels
+        .createChannel(true, randomName, randomName, json.plAddress.currency)
+        .then(channelsResp => {
+          const channelId = channelsResp.body.data.channelCreate.channel.id;
+          const channelSlug = channelsResp.body.data.channelCreate.channel.slug;
+          customer
+            .createCustomer(randomEmail, randomName, json.plAddress)
+            .then(resp => {
+              const customerId = resp.body.data.customerCreate.user.id;
```

Comment:
These could be done with 
```suggestion
              const { user: { id : customerId, email: customerEmail } } = resp.body.data.customerCreate
```